src/model/Limits.java

```diff
@@ -0,0 +1,43 @@
+package model;
+
+public class Limits {
+  private double upper;
+  private double lower;
+
+  public Limits(double upper, double lower){
+    this.upper = upper;
+    this.lower = lower;
+  }
+
+  public void setLower(double lower) throws Exception {
```

Comment:
These should be accessible from the model/modlemanager

Reply:
added